test/utils.py

```diff
@@ -40,7 +40,7 @@ class Executable(object):
     def __init__(self, exe, *before_args, **kwargs):
         self.exe = exe
         self.before_args = list(before_args)
-        self.after_args = []
+        self.after_args = kwargs.get('after_args', [])
```

Comment:
Do we still need `*before_args`?  If not we could instead just rename `before_args` to `arfter_args`?

Reply:
Hmm, I guess you're right. Done.